backend/fms_core/import_tool/handlers/experiment_run.py

```diff
@@ -0,0 +1,67 @@
+from datetime import datetime
+from pandas import pandas as pd
+
+from fms_core.models import ExperimentType, PropertyType, Instrument, Container, Sample
+from ._generic import GenericHandler
+from ...services import create_experiment_run
+
+
+
+
+class ExperimentRunHandler(GenericHandler):
+    def __init__(self, experiment_type_obj, instrument, container, start_date,
```

Comment:
Is it normal that all of this happen during the init ? The class is a big init (more or less)?

Reply:
yeah, I dont mind creating a __init__ with no args, and passing this all to a method instead (in a similar way to what im doing for the Importer) if you prefer

Reply:
This just seem a bit unusual to have most of the actions performed by a class done in its init.